daml-assistant/src/DAML/Assistant/Types.hs

```diff
@@ -80,3 +81,14 @@ data InstallTarget
     | InstallVersion SdkVersion
     | InstallPath FilePath
     deriving (Eq, Show)
+
+data Platform
+    = Unix
+    | Windows
+
+getPlatform :: Platform
```

Comment:
Can we not just use System.Info.isWindows?

Reply:
:+1: I'll do this in a separate PR.